apps/content/api/internal/reciter_list.py

```
@@ -0,0 +1,49 @@
+    publisher_id: list[int] | None = Field(
+        None,
+        q="assets__resource__publisher_id__in",
+        description="Filter by publisher ID.",
+    )
+
+
+@router.get("reciters/", response=list[ReciterOut])
+@paginate
+@ordering(ordering_fields=["name"])
+@searching(search_fields=["name", "slug"])
+def list_reciters(request: Request, filters: ReciterFilter = Query()):
```

Ruff (0.15.1)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)